release-notes/README.md

```diff
@@ -3,113 +3,7 @@
-    * reliability improvement for Kafka connector
-    * metrics and monitoring using spring boot actuator and adding prometheus support
-    * Bug fixes
-    * Dependency updates
-    
-## Older Releases
-
-* [Release 1.7](release-notes-1-7.md) - April 2020
-    * Bug fixes
-    
-* [Release 1.6](release-notes-1-6.md) - March 2020
-
-    * Audit Log Framework (ALF) technical preview
-    * Repository Explorer (TEX) 
-
-* [Release 1.5 (February 2020)](release-notes-1-5.md) 
-    * Metadata de-duplication identification and notification
-    * Data Engine OMAS Technical Preview
-    * Data Engine Proxy Server Technical Preview
-    
-* [Release 1.4 (January 2020)](release-notes-1-4.md) 
-    * Bug fixes, documentation
-    
-* [Release 1.3 (December 2019)](release-notes-1-3.md)
-    * Conformance test suite improvements
-    * Open Metadata Archives
-    
-* [Release 1.2 (November 2019)](release-notes-1-2.md)
-    * Conformance test suite
-    * Asset cataloging and consumption
-    * Governance zones and metadata security
-    * Open metadata archives
-
-* [Release 1.1 (October 2019)](release-notes-1-1.md)
-    * Multi-tenant OMAG Server Platform
-    * JanusGraph based metadata repository
-
-* [Release 1.0 (February 2019)](release-notes-1-0.md)
-    * Peer-to-peer metadata exchange
-
-
-## Future Releases
-
-Our master branch is currently taking code for all future releases.
-Many of the features are large and the teams integrate code for
-partial function as soon as it is stable and has no impact on released function.
-So you will see support for much more function than is officially released.
-This way you can monitor and feedback on future items as they are developed.
-
-The draft release notes for our future planned releases are below:
-    
-* [Release 2.0](release-notes-2-0.md) - Planned May 2020
-    * https security for server chassis
-    * IT Infrastructure cataloging maintained through DevOps pipelines
-    * API development support
-
-* [Release 2.1](release-notes-2-1.md) - Planned June 2020
-    * Data Science and AI model governance
-    * Ethics governance
-    
- * [Release 2.2](release-notes-2-2.md) - Planned July 2020
-    * Governance and Privacy Programs
-    * Licenses, certification, terms and conditions governance
-    
-* [Release 2.3](release-notes-2-3.md) - Planned August 2020
-    * Metadata Controlled Security
-    * Data Virtualization Configuration
-    
-* [Release 2.4](release-notes-2-4.md) - Planned September 2020
-    * Digital Architecture
-    * Digital service lifecycle management
-    
-* [Release 2.5](release-notes-2-5.md) - Planned November 2020
-    * Project management of IT change
-    * IoT and large scale cohorts
-
-
-## Reporting issues
-
-If you discover an issue in the release you are using, we recommend
-first upgrading to the latest available release.  If this does not
-resolve the problem, please raise a new
-[git issue](https://github.com/odpi/egeria).
```

Comment:
I am wondering why the release notes are not appended to this readme, as before. Are we moving the release notes out of github?

Reply:
When we release we remove all of the notes (including the main readme) from the branch - it's the same as 1.8, 1.7 etc.

We can open up an issue to discuss a better approach, but the change here is aligned with what we've done previously.

The release notes will be updated in master (this still needs to be done - contributions welcome - seperate PR though as targets master)